.github/workflows/code-cov.yml

```diff
@@ -0,0 +1,43 @@
+name: Run & Update Codecov
```

Comment:
@nkorinek when you are working tomorrow - you can make this fix in how code cov runs to the other repos. essentially it wasnt triggering i think because it wasn't running on PR and pushed commits. i'm not sure but this did trigger a build in your pr!